src/services/chains/solana/transactionService.ts

```diff
@@ -0,0 +1,77 @@
+import { logger } from '../../../utils/logger';
+import SolanaWeb3, { ParsedInstruction } from '@solana/web3.js';
+import {
+  NetworkTransactionInfo,
+  TransactionDetailInput,
+  validateTransactionWithInputData,
+} from '../index';
+import { normalizeAmount } from '../../../utils/utils';
+import {
+  i18n,
+  translationErrorMessagesKeys,
+} from '../../../utils/errorMessages';
+
+const SOLANA_NODE_RPC_URL = process.env.SOLANA_NODE_RPC_URL as string;
+const solana = new SolanaWeb3.Connection(SOLANA_NODE_RPC_URL);
```

Comment:
Let's not put it here, this will be executed each time the module is imported. It means the whole app will crash if anything happens to it! 
Please move it inside a function. It's not a big deal if we instantiate the connection each time we do the validation.

Reply:
Done